### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by fabaff**

### CODE REVIEW

File: community/py-batinfo/APKBUILD [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+pkgver=0.4.2
+pkgrel=0
 pkgdesc="A simple Python lib to retreive battery information"
 url="https://github.com/nicolargo/batinfo"
 arch="noarch"
 license="LGPL3+"
 depends=""
 makedepends="python2-dev py-setuptools python3-dev"
-subpackages="py3-${pkgname#py-}:_py3 py2-${pkgname#py-}:_py2"
+subpackages="py3-${pkgname#py-}:_py3 py2-${pkgname#py-}:_py2 py3-${pkgname#py-}-doc:_py3 py2-${pkgname#py-}-doc:_py2"
```
Comment (jirutka) [inline community/py-batinfo/APKBUILD]: WAT? o.O

**Comment by jirutka** [comment]
⚠️ soname difference detected:

```diff
-usr/lib/libzmq.so.5.0.1
+usr/lib/libzmq.so.5.0.2
```

@clandmeter, I’m not sure how this should be handled – rebuild all packages that depends on libzmq?

File: community/py-zmq/APKBUILD [hunk 2]
```diff
@@ -8,9 +8,7 @@ pkgdesc="Python bindings for ZeroMQ"
 url="http://pypi.python.org/pypi/pyzmq"
 arch="all"
 license="LGPL3+ BSD"
-depends=""
 makedepends="python2-dev python3-dev py-setuptools zeromq-dev"
-install=""
 subpackages="py3-${pkgname/py-/}:_py3 py2-${pkgname/py-/}:_py2"
 source="https://files.pythonhosted.org/packages/source/p/$_pkgname/$_pkgname-$pkgver.tar.gz"
 builddir="$srcdir/$_pkgname-$pkgver"
```
Comment (jirutka) [inline community/py-zmq/APKBUILD]: There’s no upgrade, this package is already 16.0.2.

**Comment by fabaff** [comment]
soname bumps requires to rebuild all depended packages.

**Comment by clandmeter** [comment]
Lets hold this until after we have created the v3.5 branch.

**Comment by jirutka** [comment]
I’ve merged py-batinfo and main/goaccess. We’ll upgrade zeromq after releasing v3.5.

**Pull request closed without merging**